New_wave3.rb

```diff
@@ -0,0 +1,104 @@
+#Create Plant class
+class Planet
+  attr_reader :planet,:desc,:orbit,:diameter,:rotate,:distance
+  def initialize(planet,desc,orbit,diameter,rotate,distance)
+    @planet = planet
+    @desc = desc
+    @orbit =orbit
+    @diameter=diameter
+    @rotate = rotate
+    @distance = distance
+  end
+  def planet_attributes
+    attributes = "Planet: #{@planet}\nDesc: #{@desc}\nOrbit: #{@orbit}\nDiameter: #{@diameter}\nYear Length: #{@rotate}\nDistance from Sun: #{@distance}\n"
+    return attributes
+  end
+end
+
+#Create Solar System Class
+class SolarSystem
+  attr_accessor :planet
+  def initialize(planet)
+    @planet = planet
+  end
+  def add_planet(new_planet)
+    @planet<<new_planet
+  end
+  def return_planets
+    list_of_planets = []
+    @planet.each do |planet|
+      list_of_planets << "#{@planet.index(planet)+1}. #{planet}"
+    end
+    return list_of_planets
+  end
+  #Calling Planet Class to print attributes from SolarSystem class
+  def check
+    choice=gets.chomp.to_i
+    if choice == 1
+      element = Planet.new("Mercury","Smallest planet","47.8km/sec","4878km","87.97days","70millionkm")
```

Comment:
Why create new planets here?  Why not simply return the `Planet` from `@planet`?  

Look at this method closely?  Think about how you could create a method that takes a number and returns the indicated planet from `@planet`.  Get help from me or a tutor/TA if you need.